lib/tagnews/geoloc/tag.py

```diff
@@ -27,9 +29,18 @@
 
 
 def post_process(geostring):
-    # TODO
-    geostring += ' Chicago, Illinois'
-    geostring.replace('block of ', '')
+    assert type(
+        addr_list) == str, "use a string! this is an error from the post_process\
+                            function in tag.py"
+    clean = re.sub('[\W]+ ', '', addr_string)
```

Comment:
`addr_string` doesn't exist and `clean` is assigned but never used, probably want to change both to `geostring`:

```
geostring = re.sub('[\W]+ ', '', geostring)
```